wgsl/index.bs

```diff
@@ -1838,7 +1838,7 @@ memory reference</dfn> is produced.
 [=Load Rule|Loads=] from an invalid reference return one of:
     * a value from any [=memory locations|memory location(s)=] of the [[WebGPU#buffers|WebGPU buffer]]
         bound to the [=originating variable=]
-    * the zero value for store type of the reference
+    * the [=zero value=] for store type of the reference
```

Comment:
Load rule only applies to atomic-free plain types, so we don't need a zero-value for anything containing an atomic, at least from this perspective.

Reply:
Maybe we should just have a more generic memory load definition then because you could get an invalid memory reference to an atomic.

```
var<workgroup> v : array<atomic<u32>, 4>;
...
let y = atomicLoad(v[4]);
```

Reply:
I filed https://github.com/gpuweb/gpuweb/issues/1868 about this.